packages/jupyter-chat/src/registry.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+import { Token } from '@lumino/coreutils';
+import { IAutocompletionCommandsProps } from './types';
+
+/**
+ * The token for the autocomplete registry, which can be provided by an extension
+ * using @jupyter/chat package.
+ */
+export const IAutocompletionRegistry = new Token<IAutocompletionRegistry>(
+  '@jupyter/chat:IAutocompleteRegistry'
+);
+
+/**
+ * The interface of a registry to provide autocompleters.
+ */
+export interface IAutocompletionRegistry {
+  /**
+   * The default autocompletion name.
+   */
+  default: string | null;
+  /**
+   * Get the default autocompletion.
+   */
+  getDefault(): IAutocompletionCommandsProps | undefined;
```

Comment:
These two properties have a similar name and it could be confusing to know what they refer to.

Should they be renamed to something else, for example `defaultName` and `defaultCompletion`?

Reply:
Thanks @jtpio, I renamed the `getDefault()` to `getDefaultCompletion()`.